thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/common/persistence/PersistenceApp.java

```diff
@@ -0,0 +1,66 @@
+package com.linkedin.thirdeye.common.persistence;
+
+import io.dropwizard.Application;
+import io.dropwizard.assets.AssetsBundle;
+import io.dropwizard.db.DataSourceFactory;
+import io.dropwizard.migrations.MigrationsBundle;
+import io.dropwizard.setup.Bootstrap;
+import io.dropwizard.setup.Environment;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Map;
+
+public class PersistenceApp extends Application<PersistenceConfig> {
+
+  static final String USAGE = "Please pass either of the arguments : \n "
+      + "1: db status <config_dir> \n 2: db dump <config_dir> \n 3: db migrate <config_dir> \n";
+
+  @Override
+  public String getName() {
```

Comment:
I'm guessing this is the new way to apply migrations. Never mind my comment above